mixin/thanos/dashboards/replicate.libsonnet

```diff
@@ -0,0 +1,58 @@
+local g = import '../lib/thanos-grafana-builder/builder.libsonnet';
+
+{
+  local thanos = self,
+  replicator+:: {
```

Comment:
`replicator?`

Reply:
@daixiang0 please look carefully on details. It's sometimes hard but details matter (:

Reply:
I miss the mixin part since they are in process indeed, sorry for pushing them :)